src/main/java/com/commercetools/sync/sdk2/products/ProductSyncOptions.java

```diff
@@ -2,37 +2,37 @@
 
 import static java.util.Optional.ofNullable;
 
-import com.commercetools.sync.commons.BaseSyncOptions;
+import com.commercetools.api.client.ProjectApiRoot;
+import com.commercetools.api.models.product.ProductDraft;
+import com.commercetools.api.models.product.ProductProjection;
+import com.commercetools.api.models.product.ProductUpdateAction;
 import com.commercetools.sync.commons.exceptions.SyncException;
```

Comment:
I modified them right away since those classes in old package needs no migration actually.
https://github.com/commercetools/commercetools-sync-java/pull/872/commits/4feeabfcd6f91e105614fa8a38fbf136573cb6f0

In this commit, those files which green in colour are newly added to sdk2 package without any changes.